tools/repl.md

```diff
@@ -161,3 +161,4 @@ readTextFileSync  readFileSync      readDir           readLink          readAllS
 | Meta-T                | Transpose words                                                                                  |
 | Meta-U                | Upper-case the next word                                                                         |
 | Meta-Y                | See Ctrl-Y                                                                                       |
+| Alt-Enter             | Insert a new line                                                                                |
```

Comment:
This actually requires `Control` key on macOS

Reply:
There are other Key combo with alt, like alt left alt right, should those also mention Control on mac ?

Reply:
Okay, then please mention that it's `Control` on mac

Reply:
what do you think about adding a footer like this:
`Note: Alt key is replaced with Control on mac os`

Reply:
Updated the key to CTRL-S